challenges/systems/challenge-1-windows.md

```diff
@@ -2,6 +2,9 @@
 
 Implement a prototype job worker service that provides an API to run arbitrary processes.
 
+These processes can be any executable program that is available on the machine running the service.
```

Comment:
Feels like this should be under the `Requirements` section.

Reply:
Perhaps, but then we'd have to duplicate it 5 times?

Reply:
We do that for the rest of the challenge requirements, seems fine to me?

Makes it easy to cover because I just scroll down to the section candidates are applying for and cover it.